tests/migrations/helpers/test_shared_helper.py

```diff
@@ -275,10 +275,10 @@ def test_get_cherrypicked_samples_sentinel_and_beckman(config):
     with patch("sqlalchemy.create_engine", return_value=Mock()):
         with patch("pandas.read_sql", side_effect=expected):
             returned_samples = get_cherrypicked_samples(config, samples, plate_barcodes)
-            assert returned_samples.at[0, FIELD_ROOT_SAMPLE_ID] == "MCM001"
-            assert returned_samples.at[1, FIELD_ROOT_SAMPLE_ID] == "MCM006"
-            assert returned_samples.at[2, FIELD_ROOT_SAMPLE_ID] == "MCM003"
-            assert returned_samples.at[3, FIELD_ROOT_SAMPLE_ID] == "MCM005"
+            assert returned_samples.at[0, FIELD_ROOT_SAMPLE_ID] == "MCM001"  # type: ignore
```

Comment:
I have noticed all of these # type: ignore appearing. Is this because they are causing errors?

Reply:
I think it's because of an upgrade in mypy which is catching more type inconsistencies. There might be a better way to resolve it with stub files or something but I usually timebox the typing issues and when time runs out, `type: ignore` appears! :P